core/api/service/state/impl/state_api_impl.hpp

```diff
@@ -33,12 +42,21 @@ namespace kagome::api {
         const boost::optional<primitives::BlockHash> &block_hash_opt)
         const override;
 
-    outcome::result<common::Buffer> getStorage(
+    outcome::result<boost::optional<common::Buffer>> getStorage(
         const common::Buffer &key) const override;
-    outcome::result<common::Buffer> getStorage(
+    outcome::result<boost::optional<common::Buffer>> getStorage(
```

Comment:
Why isn't it named `getStorageAt`?

Reply:
I had the same question, let me rename it.